scripts/deploy.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+TAG=test-deploy-branch
```

Comment:
Can we do this without setting a default on the TAG? For instance, exit process if TAG has not been set?

Reply:
Take a look, but I think that should be handled in the travis file using conditional release option in travis.  
- https://docs.travis-ci.com/user/deployment#conditional-releases-with-on

Reply:
Oh, but wait. I see line 2 shouldn't be there.